oci/private/authn.bzl

```diff
@@ -48,6 +48,11 @@ _WWW_AUTH = {
         "scope": "repository:{repository}:pull",
         "service": "token-service",
     },
+    "nvcr.io": {
+        "realm": "{registry}/proxy_auth",
+        "scope": "repository:{repository}:pull",
+        "service": "{registry}",
```

Comment:
Both `"service": "{registry}"` and `"service": "registry"` (as shown in the URL above) worked, so I chose the former to be consistent with other similar ones.